lib/commands/web.js

```diff
@@ -387,13 +387,19 @@ helpers.useAtomsElement = function (el) {
 helpers.convertElementsForAtoms = function (args = []) {
   let newArgs = [];
   for (let arg of args) {
-    if (!_.isNull(arg) && !_.isUndefined(arg.ELEMENT)) {
-      let atomsElement = this.getAtomsElement(arg.ELEMENT);
-      if (atomsElement === null) {
-        throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg.ELEMENT}'`);
+    let hasElementKey = false;
+    for (let elementKey of ['ELEMENT', 'element-6066-11e4-a52e-4f735466cecf']) {
+      if (!_.isNull(arg) && !_.isUndefined(arg[elementKey])) {
+        hasElementKey = true;
+        let atomsElement = this.getAtomsElement(arg[elementKey]);
+        if (atomsElement === null) {
+          throw new errors.UnknownError(`Error converting element ID for using in WD atoms: '${arg.ELEMENT}'`);
+        }
+        newArgs.push(atomsElement);
```

Comment:
is it possible that both keys are present and the element would be added twice?

Reply:
I forgot to take out `newArgs.push(atomsElement)` from the for-loop.